addon/components/u-edit/shared-modules/drag-and-drop.js

```diff
@@ -0,0 +1,88 @@
+import Uploader from '@upfluence/ember-upf-utils/uploader';
+import Configuration from '@upfluence/ember-upf-utils/configuration';
+
+export default class ModuleBuilder {
+  constructor(session, toast) {
+    Object.assign(this, { session, toast });
+  }
+
+  build(editor, element) {
+    return new Module(editor, element, this.session.data.authenticated.access_token, this.toast);
+  }
+}
+
+class Module {
+  constructor(editor, element, token, toast) {
+    Object.assign(this, { editor, element, token, toast });
+
+    this.editor.registerModule(this);
+    this.editor.enableDragAndDrop();
```

Comment:
@JulienVannier66 found it

Reply:
sure there is no way to get the current options from the summernote ? :(

Reply:
I change for a better solution